server/config.go

```diff
@@ -216,6 +231,15 @@ func (c *Config) Load(configFile, policySecretPath string) error {
 				return err
 			}
 		}
+
+		// attempts to load the service account credentials if the credentials are not set yet
+		if analyticsSecretPath != "" && c.Analytics.CredentialsJSON == nil {
+			c.Analytics.CredentialsJSON, err = ioutil.ReadFile(path.Join(analyticsSecretPath, ServiceAccount))
+			if err != nil { // not returning the error since it may fall back to using fluentd endpoint
+				c.Analytics.CredentialsJSON = nil
+				log.Warnf("reading analytics service account credentials: %v", err)
```

Comment:
I used Warnf() on error reading the credentials instead of returning the error because I would like it to fall back to the fluentd endpoint if possible. Let me know if you think a different log level is more appropriate. @theganyo

Reply:
Ah. What's the use case for the fallback? Is it a config mistake? Or could it be intentional?

Reply:
I was considering it as a recovery attempt from a mistake, which could be a config error or k8s secret not applied or mounted correctly.

Reply:
If it's a config mistake, I believe we should error out as the system would not be operating as intended by the config settings.